_posts/2017-06-20-meetup.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Meep! AustinJS is BACK! Meep Meep!
+author: lon
+layout: post
+categories:
+- posts
+- meetups
+when: 2017-06-20T19:30:00-05:00
+---
+
+<img alt="Beaker" src="https://upload.wikimedia.org/wikipedia/en/5/59/Beaker_%28Muppet%29.jpg" />
+
+Hey friends! AustinJS is back with an exciting dual talk from Austin's own [Tara Vancil](https://twitter.com/taravancil) and [Paul Frazee](https://twitter.com/pfrazee) about their project, [Beaker Browser][Beaker], an experimental web browser with builtin hosting and p3p web protocols. 
```

Comment:
Is "p3p" a typo? We couldn't decide if it was a typo or just the next version of p2p.